Il2CppInterop.Runtime/Injection/ClassInjector.cs

```diff
@@ -128,6 +128,13 @@ public static bool IsTypeRegisteredInIl2Cpp(Type type)
         var currentPointer = Il2CppClassPointerStore.GetNativeClassPointer(type);
         if (currentPointer != IntPtr.Zero)
             return true;
+        if (IsTypeInjectedOnly(type)) return true;
```

Comment:
This check is redundant, injected types always have their native class pointers set in the pointer store.

Reply:
I mean I just refactored the method out.

Reply:
Fair enough I'll clean most of the redundant stuff in another commit.